fetch.bs

```diff
@@ -1734,8 +1734,13 @@ not always relevant and might require different behavior.
 <hr>
 
 <p>A <a for=/>request</a> has an associated
-<dfn export for=request id=concept-request-priority>priority</dfn> (null or a
-user-agent-defined object). Unless otherwise stated it is null.
+<dfn export for=request id=concept-request-priority>priority</dfn>, which is
+"<code>high</code>", "<code>low</code>" or "<code>auto</code>". Unless stated otherwise it is
+"<code>auto</code>".
```

Comment:
We need to update HTML's existing references to this concept to use "internal priority" instead.

As far as I can tell from https://dontcallmedom.github.io/webdex/p.html#priority%40%40request%40dfn HTML is the only specification that has references (and they are more of an informative nature).

Reply:
The [matching PR for html](https://github.com/whatwg/html/pull/8470) has the change for intental-priority along with the plumbing for fetchPriority.  Would you prefer to have the internal-priority change separate?